core/src/main/resources/error/error-classes.json

```diff
@@ -99,6 +92,10 @@
   "INVALID_JSON_SCHEMA_MAPTYPE" : {
     "message" : [ "Input schema %s can only contain StringType as a key type for a MapType." ]
   },
+  "INVALID_PARAMETER_VALUE" : {
+    "message" : [ "Invalid parameter value of '%s' in '%s'. %s" ],
```

Comment:
how about `The value of parameter '%s' in %s is invalid: %s`. Then we can have something like
`The value of parameter 'key' in function 'aes_encrypt/aes_decrypt' is invalid: expects a binary value with 16, 24 or 32 bytes, but got 9 bytes.`